### ISSUE TITLE
Screenshots for different viewport sizes

### ISSUE THREAD

**Issue by jekuno** [ronen/rspec-page-regression#11]
Here's a feature request inspired by https://github.com/garris/BackstopJS
which would be a big boost for the testing of responsive templates:

It would be nice if one could configure different viewports e.g. as follows:

```
"viewports": [
    {
      "name": "phone",
      "width": 320,
      "height": 480
    }, {
      "name": "tablet_v",
      "width": 568,
      "height": 1024
    }, {
      "name": "tablet_h",
      "width": 1024,
      "height": 768
    }
  ]
```

  Using e.g. the configuration above we would then get *three* screenshots
  (1_phone.png, 1_tablet_v.png, 1_tablet_h.png) with the according dimensions
  instead of only *one* screenshot with one default dimension.
  The rspec matcher would only return true if the screenshots of ALL viewports
  would match the provided reference images.

**Comment by ronen** [comment]
Yes, that would be good.  The viewports could be configured via `RSpec::PageRegression.config`.  The matcher could have an option to select only some of the viewports (some tests might only apply to certain sizes), as well as an option to set/override the viewport configuration. 

Happy to accept a PR!
